bangazon-clumsy-penguins/BangazonSite
# Pp shopping cart

# Name - Shopping Cart Remove

## Description
Can remove Line Item from Shopping Cart

## Steps to Test

Log in as user
click "view cart" in Nav Bar
click Remove next to a line item
Cart should refresh with line Item removed

##### Link to Feature Ticket
#7
